### PR TITLE
ci: Print hint to fix auto-generated file checks

### PR DESCRIPTION
**Pull request by sayboras**
## Description

ci(gha): Print hint to fix generated file check

This commit is to print steps on how to fix github action check for
generated files. This will surely help new contributors to have
better experience and reduce feedback loop.

Suggested by: Kornilios Kourtis <kornilios@isovalent.com>

Signed-off-by: Tam Mach <sayboras@yahoo.com>

## Testing

Testing was done by one dummy change in one of the file https://github.com/cilium/cilium/pull/13718/checks?check_run_id=1296840070

```diff
@@ -690,7 +690,3 @@ spec:
       - configMap:
           name: cilium-config
         name: cilium-config-path
-
-
-
please run 'make -C install/kubernetes quick-install' and submit your changes
Error: Process completed with exit code 1.
```

```release-note
ci: Print hint to fix auto-generated file checks
```

### CODE REVIEW

**Review by kkourt — APPROVED** [review]
Thanks for doing this! :rocket:

**Comment by sayboras** [comment]
Mark ready to merge as changes are only for github action (e.g. full ci is not required).

File: .github/workflows/smoke-test.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Run helm-docs
         run: |
           make -C install/kubernetes docs
-          git diff --exit-code
+          git diff --exit-code || (echo "please run 'make -C install/kubernetes docs' and check in files"; exit 1)
```
Comment (tklauser) [inline .github/workflows/smoke-test.yaml]: Nit: all other messages say "...and submit your changes" here. Maybe change it as well to stay consistent?
```suggestion
          git diff --exit-code || (echo "please run 'make -C install/kubernetes docs' and submit your changes"; exit 1)
```

Comment (sayboras) [reply .github/workflows/smoke-test.yaml]: good pick, thanks :100:

**Review by tklauser — APPROVED** [review]

**Pull request merged**